### ISSUE TITLE
Crash on clicking anything in "Packages > Command Palette" 

### ISSUE THREAD

**Issue by dmitriy-korotayev** [avocode/avocode#179]
Title says it all - when I, for example, click on \<h1> in the opening list, I get an error.

Console log:
```
[14955:0316/211833:INFO:CONSOLE(1)] "Uncaught TypeError: Cannot read property 'replace' of undefined", source: /media/media/apps/linux/avocode/resources/app/node_modules/avocode/lib/text-commands.js (1)

(avocode:14955): libnotify-CRITICAL **: notify_notification_close: assertion 'NOTIFY_IS_NOTIFICATION (notification)' failed
[14955:0316/211839:ERROR:browser_main_loop.cc(207)] GLib-GObject: invalid uninstantiatable type '(null)' in cast to 'GObject'
[14955:0316/211839:ERROR:browser_main_loop.cc(207)] GLib-GObject: g_object_get_data: assertion 'G_IS_OBJECT (object)' failed

[1]  + segmentation fault (core dumped)  avocode
```

**Comment by dmitriy-korotayev** [comment]
Interestingly enough, things have started to work fine after 5-10 minutes after myself posting this issue. It can be closed, I guess

**Comment by petrbrzek** [comment]
Thanks for reporting. We'll definitely look at this.

**Comment by tomashanacek** [comment]
Fixed in Avocode 2.0.0
